content/en/dashboards/scheduled_reports.md

```diff
@@ -18,97 +18,83 @@ aliases:
 
 ## Permissions
 
-Only users with the **Dashboard Report Write** permission can generate a report. This permission is turned on by default for Admins and off for all other roles.
+Users need the `generate_dashboard_reports` [permission][permissions] to create and edit report schedules.
+This permission can be granted by another user with the `user_access_manage` permission and is available by default to users with the `org_management` permission.
+
+{{< img src="dashboards/scheduled_reports/dashboard_permissions.png" alt="A screenshot of an individual user's permissions from within the organization settings page. The dashboards report write permission is highlighted under the dashboards section" style="width:90%;" >}}
 
-Images generated in reports show all data regardless of granular read restrictions. Datadog recommends limiting the report permissions to users who have no granular read restrictions on data. To grant a user the **Dashboard Report Write** permission, create a new role with the **Dashboards Report Write** permission turned on and assign the user to this role. Alternatively, assign the **Admin** role to this user. To learn about managing roles and permissions, see [User Management][17]. 
+Users with the `org_management` permission can enable or disable the scheduled reports feature for their organization from the **Settings** tab under [Public Sharing][public-sharing] in **Organization Settings**.
 
-{{< img src="dashboards/scheduled_reports/dashboard_permissions.png" alt="A screenshot of an individual user's permissions from within the organization settings page. The dashboards report write permission is highlighted under the dashboards section" style="width:100%;" >}}
+{{< img src="dashboards/scheduled_reports/report_management.png" alt="The Report Management setting under the Settings tab in Public Sharing within Organization Settings in Datadog with the setting Enabled" style="width:90%;" >}}
 
-Users with the Admin role or **Org Management** permission can enable or disable the scheduled reports feature for an account from the **Settings** tab under [Public Sharing][15] in **Organization Settings**.
+## Unsupported widget types
 
-{{< img src="dashboards/scheduled_reports/report_management.png" alt="The Report Management setting under the Settings tab in Public Sharing within Organization Settings in Datadog with the setting Enabled" style="width:100%;" >}}
+The following widget types are **not** supported:
+- [Embedded App][embedded-app-widget]
+- [Experimental][experimental-widget]
+- [Iframe][iframe-widget]
+- [Image][image-widget]
+- [Hostmap][hostmap-widget]
+- [Run Workflow][run-workflow-widget]
+- [Wildcard][wildcard-widget]
```

Comment:
https://github.com/DataDog/dogweb/blob/52dc55daba8b071d82183df0ef2f61834f49e746/dogweb/model/dashboard_widget/lib/widget_manager.py#L207-L217

https://datadoghq.atlassian.net/wiki/spaces/DB/pages/3349708842/PDF+Reports+Supported+Widgets+Data+Sources

Reply:
Can you also add a sentence on what would happen if your dashboard includes one of these unsupported widgets?

Reply:
done!